### ISSUE TITLE
Visual keyboard hint can reappear using J + C shortcut

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#534]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.4

**Browser**
chrome

**Problem description**
https://github.com/phetsims/qa/issues/706

If a balloon has only been moved laterally from its original position, when the keyboard shortcut "J + C" is used, the visual keyboard hint will reappear around the balloon. If the balloon is moved up or down, then the keyboard hints do not reappear when "J + C" are pressed. 

**Steps to reproduce**
1. Using keyboard nav, move the balloon laterally to the sweater to pick up electrons. 
2. Then press the nav keys  "J + C"

**Visuals**

https://user-images.githubusercontent.com/87318828/132745765-45586a43-796b-4b5b-9d86-bc1c9d8854cc.mov

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.1/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.1 2021-09-07 23:14:39 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.63 Safari/537.36
Language: en-US
Window: 1351x659
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by Nancy-Salpepi** [comment]
On Mac 11 + chrome/safari: 
I noticed that this will only happen with the yellow balloon--with either one or two balloons present. The keyboard hints don't reappear when following the above steps with the green balloon.

**Comment by jbphet** [comment]
I don't know if this is the correct behavior or not.  It seems like the rule of thumb is that whenever the balloon is returned to its original position, the hint comes back.  I'll assign to @jessegreenberg for his input.  If this is _not_ in fact the desired behavior, I am of the opinion that it is not severe enough to block publication of the 1.5 release.

**Comment by jbphet** [comment]
We reviewed this in today's sound design meeting and agreed that this is _not_ the intended behavior.  The hint should disappear once the balloon is moved after a reset and should not reappear.

Having said that, we also feel that this should not be blocking to the 1.5 release.

**Comment by jessegreenberg** [comment]
The GrabDragInteraction in BASE has this option which is causing this specifically
```js
      showDragCueNode: () => model.positionProperty.get().equals( model.positionProperty.initialValue ),
```

This was fixed in the above commit.

I was surprised to see that the default behavior for GrabDragInteraction is to always show the "drag" cue, so if this is generally expected we will have to implement and provide this option. Ill make an issue in scenery-phet to see if this should become the default behavior.

@jbphet back to you to decide if this should be cherry picked into 1.5 or if this should just closed.

**Comment by jbphet** [comment]
Thanks for the fix @jessegreenberg.  This problem is both minor and rare, and I'd have to create a new branch in scenery-phet to move this onto the 1.5 release branch, so I'm making the call that it's not worth the time and effort.  Closing.
